File: src/main/java/kr/syeyoung/dungeonsguide/mod/whosonline/api/data/CosmeticData.java
package kr.syeyoung.dungeonsguide.mod.whosonline.api.data;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class CosmeticData {
    public int id;
    public String display;
    public String description;
    public String name;
    public int required_flags;
}

File: src/main/java/kr/syeyoung/dungeonsguide/cosmetics/data/CosmeticData.kt
package kr.syeyoung.dungeonsguide.cosmetics.data

import java.util.*

data class CosmeticData(
    val id: UUID? = null,
    val cosmeticType: String? = null,
    val reqPerm: String? = null,
    val data: String? = null,
)